CreatubblesAPIClientIntegrationTests/Spec/Gallery/GalleriesResponseHandlerSpec.swift

```diff
@@ -38,7 +38,7 @@ class GalleriesResponseHandlerSpec: QuickSpec
             {
                 let request = GalleriesRequest(page: 1, perPage: 10, sort: .popular, userId: nil, query: nil)
                 let sender =  TestComponentsFactory.requestSender
-                waitUntil(timeout: 10)
+                waitUntil(timeout: 25)
```

Comment:
maybe just add short/long (10/25) and leave at that for now? just so next time we don't have to change 10 ints lol